dv/uvm/riscv_dv_extension/ibex_log_to_trace_csv.py

```diff
@@ -31,9 +31,9 @@ def process_ibex_sim_log(ibex_log, csv):
             if re.search("ecall", line):
               break
             # Extract instruction infromation
```

Comment:
unrelated nit: infromation -> information

Reply:
Good catch, thanks for pointing it out.